### ISSUE TITLE
Password containing two %-characters breaks chart display

### ISSUE THREAD

**Issue by mbacklund** [phpmyadmin/phpmyadmin#16546]
### Describe the bug

I had two % characters in my password, and was wondering why I get an internal server error 500 when trying to display charts on my data.

### To Reproduce

1. Go to a table, select data wanted
2. Click on 'Display chart'
3. See error

### Server configuration

- Operating system: Debian 10
- Web server: Apache 2.4.38
- Database version: MariaDB 10.3.27
- PHP version: 7.3.19-1~deb10u1
- phpMyAdmin version: 5.0.2

### Client configuration

- Browser: Firefox 84.0
- Operating system: Windows 10

### Additional context
```
PHP Fatal error:  Uncaught Symfony\\Component\\DependencyInjection\\Exception\\ParameterNotFoundException: You have requested a non-existent parameter "XXX".
in /*/phpmyadmin/vendor/symfony/dependency-injection/ParameterBag/ParameterBag.php:100\n
Stack trace:\n#0 /*/phpmyadmin/vendor/symfony/dependency-injection/ParameterBag/EnvPlaceholderParameterBag.php(69): Symfony\\Component\\DependencyInjection\\ParameterBag\\ParameterBag->get('XXX')\n
#1 /*/phpmyadmin/vendor/symfony/dependency-injection/ParameterBag/ParameterBag.php(230): Symfony\\Component\\DependencyInjection\\ParameterBag\\EnvPlaceholderParameterBag->get('XXX')\n
#2 [internal function]: Symfony\\Component\\DependencyInjection\\ParameterBag\\ParameterBag->Symfony\\Component\\DependencyInjection\\ParameterBag\\{closure}(Array)\n
#3 /*/phpmyadmin/vendor/symfony/dependency-injection/ParameterBag/ParameterBag.php(219): preg_replace_callback('/%%|%([^%\\\\s]+)%...', Object(Closure), 'XXX')\n
#4 /*/h in /*/phpmyadmin/vendor/symfony/dependency-injection/ParameterBag/ParameterBag.php on line 100
```

### Hotfix

Change the password used by phpmyadmin to contain a maximum of one % character.

**Comment by williamdes** [comment]
Hi
Thank you for the report, we will have a look into it

https://github.com/code-lts/phpmyadmin-releases/blob/upstream/version/source/5.0.4/vendor/symfony/dependency-injection/ParameterBag/EnvPlaceholderParameterBag.php

**Comment by williamdes** [comment]
@mbacklund Could you try the latest 5.0 version ?

**Comment by mbacklund** [comment]
@williamdes Thanks, it looks like it's working on the latest version.

**Comment by williamdes** [comment]
> @williamdes Thanks, it looks like it's working on the latest version.

Cool, I was sure we fixed something related to that but I could not find it

**Comment by williamdes** [comment]
I found it, It is #16222
Fixed in 5.0.3
